definitions/npm/ramda_v0.26.x/flow_v0.76.x-/test_ramda_v0.26.x_relation.js

```diff
@@ -46,6 +50,50 @@ const eqb: boolean = _.eqBy(Math.abs, 5, -5);
 
 const es: boolean = _.equals([1, 2, 3], [1, 2, 3]);
 
+describe('ascend', () => {
+  /**
+   * In the examples for ascend, its result is plugged into sort. In order
+   * for this to function properly ascend needs to yield a function that can
+   * take two arguments.
+   */
```

Comment:
Awesome. I love your comments.

It might make more sense to put those in the implementation though?

Reply:
I agree!

The comment has been added to the implementation with some additional context that I think makes more sense there. I left the comments in the tests because it still feels like a surprising case that should be called out, and `it` blocks don't capture the detailed intent as well. Feel free to be very picky on what you'd like to see there though.